src/constants/script_constants.asm

```diff
@@ -278,3 +278,25 @@ WEST     EQU $03
 NO_MOVE  EQU %10000000 ; For rotations without movement
 
 VARIABLE_CARD	EQU 0 ; use the card located in wCardReceived instead of using the script's argument
+
+; medal flags set in EVENT_MEDAL_FLAGS
```

Comment:
I think this order does not match the actual order in `EVENT_MEDAL_FLAGS`. You can compare it to `EventVarMasks:` or `MedalEvents:`.
I think just Psychic and Fighting are swapped.